ha/core/controllers/pcs/cluster_controller.py

```diff
@@ -133,6 +134,25 @@ def start(self) -> dict:
             raise ClusterManagerError(f"Failed to start all nodes {failed_node_list}")
         else:
             status += "All node started successfully, resource start in progress."
+            Log.info('Cluster start is in progress. Now, checking the status of the node. standby or unstandby')
```

Comment:
This should be part of node controller
then cluster_controller is not need to be updated

Reply:
Sorry. But I feel this is the right place. The following flow explains it better:
1. unstandby the node
2. cluster stop
3. cluster start
After this, first cluster will be started (**pcs cluster start** internally). Then individual node start will be called.
In the node start, first node status will be fetched using **pcs status nodes**. And this is coming as **offline**. 
So, for this, internally **pcs cluster start srvnode-1**  is getting executed. and control will be back to **cluster_executor** start method (from where it started). 
response is : 
 [start] res: {'status': 'InProgress', 'msg': 'Node srvnode-1 : Node was in cluster_offline mode, Cluster started on node successfully'}

So,  I think, its the right position to check this condition. 
Also, instead of performing unstandby for all node one by one, we can perform unstandby using **--all** flag.

Let me know if still there is a point of discussion. 
Thanks !

Reply:
Also, instead of performing unstandby for all node one by one, we can perform unstandby using --all flag.

We need to avoide --all flag
it will face problem in 32 node

Reply:
Ok.

Reply:
As per the discussion, moving this code to node_controller so as to avoid one more loop and keep cluster controller separated from this node logic